homeassistant/components/tessie/coordinator.py

```diff
@@ -79,7 +79,7 @@ def _flattern(
         result = {}
         for key, value in data.items():
             if parent:
-                key = f"{parent}-{key}"
+                key = f"{parent}_{key}"
             if isinstance(value, dict):
                 result.update(self._flattern(value, key))
             else:
```

Comment:
Please adjust the entity/coordinator/sensor in a separate PR. Once we merge that one, this one should just be about binary sensors

Reply:
This was feedback from @joostlek. I can seperate that out when I get back in a few days, so ill have to throw all the current PRs into draft.

Reply:
I agree with his comments. It should still be a separate PR.

Reply:
Finally been able to get that changed raised seperately:
https://github.com/home-assistant/core/pull/105824